### DISCUSSION TITLE
How to add `not equal to` as validation

### DISCUSSION THREAD

**Discussion by roylisto** [cue-lang/cue#1238] [Q&A] [answered]
Does cue has something like `value: !"N.A"`? 
the context is I don't want any N.A value or similar in my configuration

**Comment by slewiskelly** [comment, marked as answer]
```cue
value: != "N.A"

value: "N.A" // value: invalid value "N.A" (out of bound !="N.A")
```

For similar, you can specify a regex with `!~`:

```cue
value: !~ #"N\.A.*"#

value: "N.Afoo" // value: invalid value "N.Afoo" (out of bound !~"N\\.A.*")
```

https://cuelang.org/docs/references/spec/#operators-and-punctuation

**Reply by roylisto** [reply]
Thank you @slewiskelly
